src/packages/auth/src/authentication/apollo/plugin.ts

```diff
@@ -227,7 +227,7 @@ export const authApolloPlugin = <R>(
 
 						upsertAuthorizationContext(contextValue);
 					} catch (err: unknown) {
-						logger.trace(`JWT verification failed. ${err}`);
+						logger.error(`JWT verification failed. ${err}`);
```

Comment:
Can we log the whole error instead of stringifying it, e.g.
```suggestion
						logger.error({ err }, `JWT verification failed.`);
```

Reply:
Yeah good call, I've made that change locally to check the output and pushed it up